kpi/models/asset.py

```diff
@@ -542,6 +544,16 @@ def get_advanced_submission_schema(self, url=None, content=False):
             content, self.advanced_features, url=url
         )
 
+    @cache_for_request
+    def get_attachment_xpaths(self, deployed: bool = True) -> list:
+        v = self.latest_deployed_version if deployed else self.latest_version
+        survey = v.to_formpack_schema()['content']['survey']
```

Comment:
I can mostly tell what this function is doing right now, so I don't think it's a huge deal, but adding (brand new) one-letter variables is a little scary. 😅